pkg/registry/core/service/strategy.go

```diff
@@ -101,6 +101,7 @@ func (strategy svcStrategy) PrepareForCreate(ctx context.Context, obj runtime.Ob
 	}
 
 	dropServiceDisabledFields(service, nil)
+	dropTypeDependentFields(service, nil)
```

Comment:
would you envision ipFamilies/ipFamilyPolicy/clusterIPs population happening *after* this call?

Reply:
Yes, allocation happens later